### ISSUE TITLE
--lua-version does nothing

### ISSUE THREAD

**Issue by raingloom** [keplerproject/luarocks#510]
I'm trying to install Luarocks for OpenResty on Arch Linux. There is an installer in the AUR, but it is orphaned. I'm trying to use that as a base.

The build script is
```
  cd "$srcdir/$_pkgname-$pkgver"
  ./configure --prefix=/opt/openresty/luajit \
    --with-lua=/opt/openresty/luajit/ \
    --lua-suffix=jit-2.1.0-beta1 \
    --force-config \
    --lua-version=5.1 \
    --with-lua-include=/opt/openresty/luajit/include/luajit-2.1
  make
```
Then, I want to install Lapis, but the query returns no results. `./luarocks config --lua-ver` returns `5.3`.

**Comment by hishamhm** [comment]
Hi! Some questions to help troubleshoot this:
* What is the output of the configure script?
* How did you run make?
* I notice you're running `./luarocks` from the current directory: where? And what happens when you run `luarocks` installed in your $PATH?

**Comment by raingloom** [comment]
Hi, thanks for the quick reply!

Well, all of this is now irrelevant >.>

As I was checking out the build script I tried running `make build` (as instructed by the manual) instead of just `make` (which is what the original author of the script left over) and Luarocks now seems properly configured! Turns out I simple didn't RTFM hard enough...

As for why `./luarocks`: this is actuallly `/opt/openresty/luajit/bin/luarocks`
I have a separate one there because it is specific to Openresty only and I didn't want to confuse it with the one in my $PATH. (which is the 5.3 one I use for everything else)
